src/common/user-store/user-store.ts

```diff
@@ -81,6 +83,11 @@ export class UserStore extends BaseStore<UserStoreModel> /* implements UserStore
    */
   hiddenTableColumns = observable.map<string, ObservableToggleSet<string>>();
 
+  /**
+   * Monaco editor configs
+   */
+   @observable editorConfiguration:EditorConfiguration = {tabSize: null, miniMap: null, lineNumbers: null};
```

Comment:
If there are three different `EditorType`'s why so we only have one here?

Reply:
We decided do not to give control on each editor type to the user. Basically, this class stores global variables overriding default values.

Reply:
That seems confusing, either we should have a single config for all (which we don't because some of the defaults are different by the "Type" of tab or we should allow configuring each type separately (as you did previously).

Reply:
No I was meaning why have the different editor types at all? For a simpler initial we could just use all the same.

Reply:
Yes, it was decided because same preferences doesn't look good in every places. For example, minimap isnt look good on small editors... But thinking about it more maybe it's really better to stick with same values for every editor type? Let it just show minimap everywhere along with line numbers? @pashevskii @Nokel81

Reply:
I think that would be best, at least for the first version.